### PR TITLE
Correct some errors

### PR DESCRIPTION
**Pull request by jimjyang**
Hopefully this version will work.

### CODE REVIEW

File: demo-resources/.htaccess [hunk 1]
```diff
@@ -5,41 +5,62 @@ Options -MultiViews
...
+# Norwegian Nynorsk
+RewriteCond %{HTTP:Accept-Language} nn [NC]
+RewriteRule ^(demo-models)$ https://jimjyang.github.io/showroom/modelldcat-ap-no/files/$1-nb  [R=303,L]
+# Norwegian Norsk
+RewriteCond %{HTTP:Accept-Language} no [NC]
+RewriteRule ^(demo-models)$ https://jimjyang.github.io/showroom/modelldcat-ap-no/files/$1-nb  [R=303,L]
 # English
 RewriteCond %{HTTP:Accept-Language} en [NC]
-RewriteRule ^demo-models# https://jimjyang.github.io/showroom/modelldcat-ap-no/files/demo-models-en#  [R=302,L]
+RewriteRule ^(demo-models)$ https://jimjyang.github.io/showroom/modelldcat-ap-no/files/$1-en  [R=302,L]
```
Comment (dgarijo) [inline demo-resources/.htaccess:61]: In https://github.com/jimjyang/showroom/tree/main/modelldcat-ap-no/files you have no files ended in `en`, so this redirection will fail. Maybe you mean https://jimjyang.github.io/showroom/modelldcat-ap-no/files/demo-models-en.html#$1 ?

**Comment by dgarijo** [comment]
No, it won't because the target link does not seem to exist. Please see my comment for a suggested fix

File: demo-resources/.htaccess (hunk 1 above)
Comment (jimjyang) [reply demo-resources/.htaccess:61]: @dgarijo Thanks for your comment! 

https://jimjyang.github.io/showroom/modelldcat-ap-no/files/demo-models-en does work. Or, what did you mean?

Comment (dgarijo) [reply demo-resources/.htaccess:61]: Mhh, my bad, it looks like I tested it out the wrong file. Apologies!

Comment (jimjyang) [reply demo-resources/.htaccess:61]: Thanks!

**Pull request merged**